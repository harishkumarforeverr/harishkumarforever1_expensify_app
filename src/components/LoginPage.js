import React from "react";
import {connect} from "react-redux";
import {startLogin} from "../actions/auth";

export const LoginPage=(props)=>{
    return (
        <div>
            <button onClick={props.startLogin}>Login</button>
        </div>
    )
}

const mapDispatchToState=(dispatch)=>{
    return {
        startLogin:()=>{dispatch(startLogin())}
    }
}

export default connect(undefined,mapDispatchToState)(LoginPage);