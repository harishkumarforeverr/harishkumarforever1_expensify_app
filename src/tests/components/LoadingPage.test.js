import React from "react"
import{shallow} from "enzyme";
import LoginPage from "../../components/LoadingPage";

test("LoadingPage.test  should match snapshot ",()=>{
     const wrapper=shallow(<LoginPage/>);
     expect(wrapper).toMatchSnapshot();
});