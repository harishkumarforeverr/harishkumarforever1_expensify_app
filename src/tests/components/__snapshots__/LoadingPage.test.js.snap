// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage.test  should match snapshot  1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
